src/docs/stan-reference/appendices.tex

```diff
@@ -803,6 +805,34 @@ \subsection{Probability Function Naming}
 A probability function literal must have one of the following
 suffixes: \code{\_lpdf}, \code{\_lpmf}, \code{\_lcdf}, or \code{\_lccdf}.
 
+\subsection{Algebraic Solver Argument Types and Origins}
+
+The \code{algebra\_solver} function may be used without control
+parameters; in this case
+%
+\begin{itemize}
+\item its first argument refers to a function with signature
+\begin{quote}
+  \code{(~real[], ~real[],~real[],~int[]~:~real[]},
```

Comment:
The semicolon is not part of the code, but of the sentence in which the code is quoted.

Same syntax was used for integrate_ode_*, line 844:
```
\begin{itemize}
\item its first argument to refer to a function with signature
\begin{quote}
 \code{(real,~real[],~real[],~real[],~int[])~:~real[]},
\end{quote}
```